doc/getting-started/building-the-node-using-nix.md

```diff
@@ -24,15 +24,16 @@ To improve build speed, it is possible to set up a binary cache maintained by IO
 optional**):
 ```
 sudo mkdir -p /etc/nix
-cat <<EOF | sudo tee /etc/nix/nix.conf
+cat <<EOF | sudo tee -a /etc/nix/nix.conf
```

Comment:
Can we fix the caches in this file?;we no longer use iohk.cachix.org. and the substituter should be cache.iog.io now (but the key still has hydra in the name)

Reply:
Ok, I think I understood correctly. I updated the commit.